### PR TITLE
chore: increase wasi memory limits

### PR DESCRIPTION
**Pull request by Brooooooklyn**

### CODE REVIEW

**Comment by Brooooooklyn** [comment]
* **#2035** <a href="https://app.graphite.dev/github/pr/napi-rs/napi-rs/2035?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Brooooooklyn and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: cli/src/api/templates/load-wasi-template.ts [hunk 1]
```diff
@@ -24,8 +24,10 @@ const __wasi = new __WASI({
 const __emnapiContext = __emnapiGetDefaultContext()
 
 const __sharedMemory = new WebAssembly.Memory({
-  initial: 1024,
-  maximum: 10240,
+  // 1Gb
+  initial: 16384,
```
Comment (toyobayashi) [inline cli/src/api/templates/load-wasi-template.ts:28]: The `initial` size means `new WebAssembly.Memory()` will allocate 1GB size `SharedArrayBuffer` immediately, I personally don't think most of normal wasm module need so much initial memory, it just grow within `maximum` on demand. If there are multiple napi-rs modules are using in a same web page or a StackBlitz Node.js worker, is this too waste?

File: cli/src/api/templates/load-wasi-template.ts [hunk 2]
```diff
@@ -95,8 +97,10 @@ const __wasi = new __nodeWASI({
...
```
Comment (toyobayashi) [inline cli/src/api/templates/load-wasi-template.ts:101]: Same here

File: cli/src/api/templates/load-wasi-template.ts (hunk 1 above)
Comment (Brooooooklyn) [reply cli/src/api/templates/load-wasi-template.ts:28]: Good point. Let's make it configurableble

**Pull request merged**